.github/workflows/ci.yml

```diff
@@ -656,7 +657,7 @@ jobs:
           yum install tar gzip which -y
       - uses: actions/checkout@v4
       - name: "Install Python"
-        run: yum install python3 python3-pip -y
+        run: yum install python3 -y && python3 -m ensurepip
```

Comment:
`pip` installed via `yum install python3-pip` has `RECORD` missing, which causes issues described in https://github.com/pypa/pip/issues/11631